src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -194,6 +200,26 @@ protected void updateItem(final TableRow item, final boolean empty) {
       rootItem.setExpanded(true);
    }
 
+   private void collapseExpandTable(){
+
+      if(expanded){
+         //CollapseAll
+         expand(false);
+         expandCollapseButton.setText("Expand");
+
+      }else {
+         //ExpandAll
```

Comment:
do we need the comments if we rename the method?

Reply:
deleted them